examples/error-type-intersection/error_type_intersection.metatags

```diff
@@ -0,0 +1,2 @@
+description: This BBE demonstrates type intersection for error types.
+keywords: ballerina, ballerina by example, bbe, error, intersection, distinct
```

Comment:
Do we need `distinct` keyword in here

Reply:
It's OK to have IMO. This is pretty much the only way to define subtypes of a distinct error type with additional constraints.